.github/workflows/ci-dockers.yml

```diff
@@ -44,9 +44,9 @@ jobs:
           # We only release one docker image per PyTorch version.
           # The matrix here is the same as the one in release-docker.yml.
           - {python_version: "3.9", pytorch_version: "1.11", cuda_version: "11.3.1"}
-          - {python_version: "3.9", pytorch_version: "1.12", cuda_version: "11.6.1"}
-          - {python_version: "3.9", pytorch_version: "1.13", cuda_version: "11.7.1"}
-          - {python_version: "3.10", pytorch_version: "2.0", cuda_version: "11.7.1"}
```

Comment:
@Borda This item should be added back. It's the image used in Fabric and PL Azure GPU CI. Alternatively you can update it to use the ones you added.